docs/internal/pysnmp_debug.md

```diff
@@ -0,0 +1,33 @@
+# PYSNP debug mode (internal)
+
+During on-calls and escalations might be helpful to understand low-levels details that happening on SNMP protocol side. For that we using `debug logger` from `pysnmp lib`
```

Comment:
During the on-calls and escalations, it might be helpful to understand low-levels details that are happening on SNMP protocol side. For that we are using `debug logger` from `pysnmp lib`.